processor/transformer/transformer.go

```diff
@@ -354,7 +371,11 @@ func (trans *HandleT) request(url string, data []TransformerEventT) []Transforme
 	var transformerResponses []TransformerResponseT
 	if resp.StatusCode == http.StatusOK {
 		integrations.CollectIntgTransformErrorStats(respData)
-		err = json.Unmarshal(respData, &transformerResponses)
+
+		trace.Logf(ctx, "Unmarshal", "response raw size: %d", len(respData))
```

Comment:
Do we need these trace logs in Production?

Reply:
These will help analysing traces.